configure

```diff
@@ -278,15 +278,15 @@ if [ -n "$NETCDF4" ] ; then
     if [ -f $NETCDF/bin/nf-config ] ; then
       nx_config="$NETCDF/bin/nf-config --flibs"
       DEP_LIB_PATH="`$nx_config | awk '{for(i=1;i<=NF;i++){if(match($i, /-L.*/)) {print $i} } }'`"
-      CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lcurl") {print $i} } }'`"
-      GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i == "-lgpfs") {print $i} } }'`"
+      CURL="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lcurl") {print $i} } }'`"
+      GPFS="`$nx_config | awk '{for(i=1;i<=NF;i++){if($i = "-lgpfs") {print $i} } }'`"
```

Comment:
I may be mistaken, but the if-test here is for `awk` and not `sh`, so I don't think these should be modified.